app/src/main/java/com/x8bit/bitwarden/data/platform/datasource/disk/SettingsDiskSourceImpl.kt

```diff
@@ -37,6 +37,7 @@ private const val SHOW_AUTOFILL_SETTING_BADGE = "showAutofillSettingBadge"
 private const val SHOW_UNLOCK_SETTING_BADGE = "showUnlockSettingBadge"
 private const val SHOW_IMPORT_LOGINS_SETTING_BADGE = "showImportLoginsSettingBadge"
 private const val LAST_SCHEME_CHANGE_INSTANT = "lastDatabaseSchemeChangeInstant"
+private const val IS_VAULT_REGISTERED_FOR_EXPORT = "isVaultRegisteredForExport"
```

Comment:
Make sure to add this to the `clearData` function.

Reply:
Nice catch. Thank you!